curriculum/challenges/english/20-upcoming-python/learn-tree-traversal-by-building-a-binary-search-tree/655a227e57aabb25d1f9c987.md

```diff
@@ -0,0 +1,59 @@
+---
+id: 655a227e57aabb25d1f9c987
+title: Step 19
+challengeType: 20
+dashedName: step-19
+---
+
+# --description--
+
+Now, it's time to work on the search functionality.
+
+Create a method named `search` inside the `BinarySearchTree` class. Give the `search` method two parameters: `self` and `key`.
```

Comment:
```suggestion
Define a method named `search` inside the `BinarySearchTree` class. Give the `search` method two parameters: `self` and `key`.
```
I would suggest replacing "create" with "define" to be consistent with the hint.